src/pages/Admin/Admin.tsx

```diff
@@ -427,7 +427,7 @@ const Admin: React.FC = () => {
 
   const { isAdminUser, isLoading: isAdminUserLoading } = useIsAdminUser(userId);
 
-  const venuesOwnedByUserQuery = useMemo<ReduxFirestoreQuerySetting>(
+  const venuesOwnedByUserQuery = useMemo<SparkleRFQSetting>(
     () => ({
       collection: "venues",
       where: [["owners", "array-contains", userId]],
```

Comment:
We don't actually need to `useMemo` our query configs as it does a deep compare on them anyway.